app/classifier/drawing-tools/column-rectangle.cjsx

```diff
@@ -40,20 +42,27 @@ module.exports = React.createClass
 
   initCoords: null
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-column'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-column'
+
   render: ->
     {x, width} = @props.mark
 
     <DrawingToolRoot tool={this}>
-      <Draggable onDrag={@handleMainDrag} onEnd={deleteIfOutOfBounds.bind null, this} disabled={@props.disabled}>
+      <Draggable onDrag={@handleMainDrag} onEnd={@dragEnd.bind this, this} disabled={@props.disabled}>
```

Comment:
Thinking of maintainability, is there anything we could do to make `@dragEnd.bind this, this` a bit clearer? Hard to understand why it's passing the same param twice...